[2024-07-21T09:53:41.049+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-21T09:53:41.320+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: churn_model_DAG.model_training manual__2024-07-21T09:53:29.719077+00:00 [queued]>
[2024-07-21T09:53:41.341+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: churn_model_DAG.model_training manual__2024-07-21T09:53:29.719077+00:00 [queued]>
[2024-07-21T09:53:41.343+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-21T09:53:41.363+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): model_training> on 2024-07-21 09:53:29.719077+00:00
[2024-07-21T09:53:41.377+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=2158) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-21T09:53:41.379+0000] {standard_task_runner.py:64} INFO - Started process 2167 to run task
[2024-07-21T09:53:41.379+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'churn_model_DAG', 'model_training', 'manual__2024-07-21T09:53:29.719077+00:00', '--job-id', '479', '--raw', '--subdir', 'DAGS_FOLDER/model_pipeline_DAG.py', '--cfg-path', '/tmp/tmpqvp6qgp0']
[2024-07-21T09:53:41.382+0000] {standard_task_runner.py:91} INFO - Job 479: Subtask model_training
[2024-07-21T09:53:41.431+0000] {task_command.py:426} INFO - Running <TaskInstance: churn_model_DAG.model_training manual__2024-07-21T09:53:29.719077+00:00 [running]> on host 6e10856c4017
[2024-07-21T09:53:41.575+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='churn_model_DAG' AIRFLOW_CTX_TASK_ID='model_training' AIRFLOW_CTX_EXECUTION_DATE='2024-07-21T09:53:29.719077+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-21T09:53:29.719077+00:00'
[2024-07-21T09:53:41.580+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-21T09:53:42.572+0000] {model_train.py:67} INFO - La tracking URI de MLflow tracking URI es: file:/opt/***/dags/data/mlruns
[2024-07-21T09:53:42.611+0000] {model_train.py:71} INFO - El nombre del experimento es: RandomForestClassifier
[2024-07-21T09:53:42.727+0000] {logging_mixin.py:188} WARNING - 2024/07/21 09:53:42 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2024-07-21T09:57:51.314+0000] {logging_mixin.py:188} INFO - Run completed!
[2024-07-21T09:57:51.316+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-21T09:57:51.319+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-21T09:57:51.342+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=churn_model_DAG, task_id=model_training, run_id=manual__2024-07-21T09:53:29.719077+00:00, execution_date=20240721T095329, start_date=20240721T095341, end_date=20240721T095751
[2024-07-21T09:57:51.380+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-21T09:57:51.400+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-21T09:57:51.401+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
